### ISSUE TITLE
False positive for variable-length argument lists

### ISSUE THREAD

**Issue by marcospassos** [thecodingmachine/phpstan-strict-rules#42]
The rule does not consider the type of variable-length argument lists:

```php
class Foo {
	public function foo(Bar ...$bars){
	}
}
```

```
In method "Foo::foo", parameter $bars type is "array". Please provide a @param annotation to further specify the type of the array. For instance: @param int
```
